etna/metrics/functional_metrics.py

```diff
@@ -1,14 +1,33 @@
+from enum import Enum
```

Comment:
Can we also vectorize interval metrics?

Reply:
There are some problems with that. First of all, interval metrics expect two columns to compute: lower quantile and higher quantile. It doesn't suit our `MetricFunctionSignature` and our current implementation of `Metric.__call__`. But there are few workarounds:
- Modify current implementation of interval metrics and not think about `Metric` and `MetricFunctionSignature`.
- Rethink `MetricFunctionSignature` and `Metric` to support more general cases of metrics.